src/types/questionnaire/diagnosis.ts

```diff
@@ -39,8 +39,8 @@ export interface Diagnosis {
   onset?: Onset;
   recorded_date?: string;
   note?: string;
-  created_by: UserBase;
-  updated_by: UserBase;
+  created_by?: UserBase;
+  updated_by?: UserBase;
```

Comment:
You can't make it optional, you can extend the type based on your use case for example CreateDiagnosis, EditDiagnosis etc

Reply:
Got it 👍 , but Vignesh said that `created_by` and `updated_by` will be made available only in retrieve spec, so can we move these out of base interface